docs/source/contributing/setup.rst

```diff
@@ -137,7 +135,7 @@ To simplify testing of JupyterHub, it’s helpful to use
 authenticator and SimpleLocalProcessSpawner instead of the default spawner.
 
 There is a sample configuration file that does this in
-``testing/jupyterhub_config.py``. To launch jupyterhub with this
+``testing/jupyterhub_config.py``. To launch Jupyterhub with this
```

Comment:
```suggestion
``testing/jupyterhub_config.py``. To launch JupyterHub with this
```

Reply:
@Goodiec, no worry, it's hard to spot a small difference of only one letter in these diffs.

`JupyterHub` has the upper letter `H` in the middle of the name, from `Hub`. 
Min's suggestion is to change  the name to `JupyterHub` (notice the upper letter `H`). In your commit it's currently `Jupyterhub` with the small letter `h`.

Reply:
Thank you @GeorgianaElena, I've made the correction.

Reply:
> Also, is it Ok if I help review the PRs of others?

Of course, that would be great! Thanks @Goodiec